R/add_risktable.R

```diff
@@ -26,7 +26,8 @@ add_risktable <- function(gg, ...){
 }
 
 #' @inheritParams get_risktable
-#'
+#' @param table.height A numeric value (in the range [0 - 1]) indicating the overall height of
```

Comment:
It is a relative parameter between the height used by the plot versus the height used by the tables. I would refer here to the cowplot parameter rel_heights as well.